modules/ixBidAdapter.js

```diff
@@ -285,24 +285,26 @@ function getBidRequest(id, impressions) {
  * From the userIdAsEids array, filter for the ones our adserver can use, and modify them
  * for our purposes, e.g. add rtiPartner
  * @param {array} allEids userIdAsEids passed in by prebid
+ * @param {object} flocId flocId passed in by prebid
  * @return {object} contains toSend (eids to send to the adserver) and seenSources (used to filter
  *                  identity info from IX Library)
  */
-function getEidInfo(allEids) {
+function getEidInfo(allEids, flocData) {
   // determines which eids we send and the rtiPartner field in ext
-  var sourceRTIMapping = {
+  const sourceRTIMapping = {
     'liveramp.com': 'idl',
     'netid.de': 'NETID',
     'neustar.biz': 'fabrickId',
     'zeotap.com': 'zeotapIdPlus',
-    'uidapi.com': 'UID2'
+    'uidapi.com': 'UID2',
+    'adserver.org': 'TDID'
   };
-  var toSend = [];
-  var seenSources = {};
+  let toSend = [];
+  let seenSources = {};
   if (utils.isArray(allEids)) {
-    for (var i = 0; i < allEids.length; i++) {
+    for (let i = 0; i < allEids.length; i++) {
```

Comment:
You can use a for-of loop here since you are only ever using i to access allEids[i]